docs/USAGE.md

```diff
@@ -0,0 +1,137 @@
+        &fs.PathExistsRequest{
+            Path: mount.NormalizeWindowsPath(path),
+        })
+    if err != nil {
+        return false, err
+    }
+    return isExistsResponse.Exists, err
+}
+```
+
+### Deployment
+
+CSI driver containers need to run as HostProcess pods for the CSI Proxy commands to complete privileged operations.
+This can be done by updating the driver pod spec, typically embedded in a Daemonset.
+The Kubernetes Windows HostProcess Pod [docs](https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/) goes into more detail about each field.
+
+```yaml
+spec:
+  securityContext:
+    hostNetwork: true
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\SYSTEM"
+```
+
+Using HostProcess pods have a few important consequences on the deployed containers, as noted in the HostProcess docs.
+- HostProcess containers have no file system or resource isolation, so they have a complete view of the host machine’s file system.
+This means that paths passed in **arguments need to be complete host machine paths**.
+On the other hand, depending on the containerd and the Windows OS version, Kubernetes volume mounts are either mounted relative to a subdirectory of the host machine specified by an environment variable `%CONTAINER_SANDBOX_MOUNT_POINT%` or **mounted relative to the host process root**. See [HostProcess Caveats](#hostprocess-caveats).
+- HostProcess pods can only contain HostProcess containers.
+Often, the CSI node registrar is deployed in the same pod as the driver, so file paths for both containers need to be updated.
+- Named pipes and Unix domain sockets are not supported and should be accessed via their path on the host.
+
+Instead of mounting host process paths such as `c:\var\lib\kubelet` to each container’s own filesystem, the container can directly access these paths.
+
+If drivers are migrating from CSI Proxy v1, note that since the current version (v2) no longer has a separate binary running on the host, drivers no longer need to mount named pipes for each API group/version used. Related volumes and volume mounts can be safely deleted.
+
+Here is an example driver's deployment configuration.
+
+```yaml
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-storage-node-win
+spec:
+  selector:
+    matchLabels:
+      app: csi-driver-win
+  template:
+    metadata:
+      labels:
+        app: csi-driver-win
+    spec:
+      serviceAccountName: csi-node-sa
+      tolerations:
+      - key: "node.kubernetes.io/os"
+        operator: "Equal"
+        value: "win1809"
+        effect: "NoSchedule"
+      nodeSelector:
+        kubernetes.io/os: windows
+      containers:
+        - name: csi-driver-registrar
+          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
+          args:
+            - --v=5
+            - --csi-address=unix://c:\var\lib\kubelet\plugins\pd.csi.storage.gke.io\csi.sock
```

Comment:
let's use placeholders for the CSI Driver instead of using PD CSI Driver as an example